src/main/java/com/connexta/store/adaptors/S3StorageAdaptor.java

```diff
@@ -110,18 +121,25 @@ public StorageAdaptorRetrieveResponse retrieve(@NotBlank final String key)
     }
   }
 
+  @Override
+  public void updateStatus(@NotBlank String key, @NotBlank String status) {
```

Comment:
❓ Can these be moved inside the if s3ObjectExists check?

Reply:
Sure, I'll move them in